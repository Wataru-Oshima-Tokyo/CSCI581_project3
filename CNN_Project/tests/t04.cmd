 input_1C_2f_2.txt weights_1C2f_2.txt structure_1C2f_2.txt 0.01 
