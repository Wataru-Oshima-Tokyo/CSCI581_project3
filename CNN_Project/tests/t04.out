0.7629642134640099 0.3527438418234110 0.8474836831387062 0.9144326474050334 0.9966982650219126 0.0002815271783269 0.2359524002025340 0.0005327102768537 0.9999971494621168 
0.4029953208617525 0.6959318743437266 0.3619293469651366 0.2819114248166502 0.1652048701906146 0.9986683447247396 0.7132047500887643 0.9887935936781571 0.0006163986844695 
